This include is necessary to get PT_NIP on musl.
--- linux-user/host/ppc64/hostdep.h
+++ linux-user/host/ppc64/hostdep.h
@@ -17,6 +17,8 @@
 
 #ifndef __ASSEMBLER__
 
+#include <asm/ptrace.h>
+
 /* These are defined by the safe-syscall.inc.S file */
 extern char safe_syscall_start[];
 extern char safe_syscall_end[];
